libs/application/templates/health-insurance/src/forms/messages.ts

```diff
@@ -116,6 +116,16 @@ export const m = defineMessages({
     defaultMessage: 'What is you current status?',
     description: 'What is you current status?',
   },
+  statusEmployed: {
+    id: 'hi.application:status.Employed',
+    defaultMessage: 'Employed',
+    description: 'Employed',
+  },
+  statusEmployedInformation: {
+    id: 'hi.application:status.Employed',
```

Comment:
Duplicate id as above. Use `employed.information` instead to follow the previous status ids  :D